### DISCUSSION TITLE
Thảo luận tuần 5+6

### DISCUSSION THREAD

**Discussion by GiaHan06** [GiaHan06/Lop-10A02-Nhom-02#22] [Ideas]
Sắp xong ròi nên mn cố lên nha!

**Comment by GiaHan06** [comment]
https://docs.google.com/document/d/1X5e6ERdfA36aRa66GnxYSn_3oVNG6bhHt2H18kdFVhs/edit?usp=sharing
đây nè tui mới xong được 80% thoi còn thiếu phần đoàn kết nữa do làm hong kịp, mấy bà xem trước mấy phần kia ok chưa ha rồi để tui cố hoàn thành hết trong hôm nay luôn

**Reply by chaudambaotram** [reply]
Có cần tìm thông tin ko? T tìm phụ cho

**Reply by GiaHan06** [reply]
ê có á m có gì bỏ vô project nha
mà tutu t chưa tạo project được máy lag quá

**Comment by NguyenNgocQuynhAnh** [comment]
@GiaHan06 xinh gái à :))

**Comment by NguyenNgocQuynhAnh** [comment]
có cần phụ gì hok chứ chưa thấy m rep nè

**Comment by NguyenNgocQuynhAnh** [comment]
hok bíc đc kh nhma t mới đưa phần kb lên í

**Comment by NguyenNgocQuynhAnh** [comment]
https://1drv.ms/w/s!AjXJGuEVMjAMhXfZxo3Bmt2npSxc?e=0G707G
này là t có thêm kb zo í đọc thử xem sao ha

**Reply by GiaHan06** [reply]
@HuynhLeBaoTran đọc chưa bà, sao hong thấy bà có ý kiến gì zạ

**Comment by NguyenNgocQuynhAnh** [comment]
ủa template là mình lấy template cho word hay sao á

**Reply by LTGHan** [reply]
Đr

**Reply by GiaHan06** [reply]
ê nãy t kím thấy cái kia ưng quá trời mà phải đk gì tùm lum á :'( còn lại hong có nào ok lắm

**Reply by GiaHan06** [reply]
nên t định tự design luôn cho ló troáy :3

**Comment by NguyenNgocQuynhAnh** [comment]
là mình phải lên mấy trang template word hả tại t thấy mấy mẫu đó hok đẹp =((

**Reply by GiaHan06** [reply]
nãy t kím trên mấy cái web ròi nhma khong có gì là miễn phí cả :'( bắt tạo tk ròi các thứ phức tạp lắm

**Comment by GiaHan06** [comment]
@NguyenNgocQuynhAnh @LTGHan muốn tự design khưm cho ló troáy :>

**Reply by NguyenNgocQuynhAnh** [reply]
desgin web nào ă m

**Reply by NguyenNgocQuynhAnh** [reply]
đc thì tự design luôn chứ mấy template xấu vải :))

**Reply by LTGHan** [reply]
Xấu vải +1

**Reply by GiaHan06** [reply]
> desgin web nào ă m

trên word luôn á chứ thầy bảo chỉ được làm trên word thoi

**Comment by GiaHan06** [comment]
@LTGHan ê m nói cái kb của  m sao kì đâu??

**Reply by LTGHan** [reply]
Tối t thấy nó kì quá nên xóa note gòi còn đâu :))

**Reply by LTGHan** [reply]
Mà z là hết thân bài ch á?

**Reply by GiaHan06** [reply]
nội dung là hoàn thành xong hớt gòi
mà m đọc thử coi có bất ổn hok

**Reply by LTGHan** [reply]
T đọc thấy ok, nma có dài quá hơm :(

**Reply by GiaHan06** [reply]
4 trang thoi chứ nhiu :(
thêm bìa với trang cuối nữa sương sương mây mây 6 trang

**Reply by LTGHan** [reply]
Gke v s

**Reply by GiaHan06** [reply]
tối thiểu 4 trang trở lên mà

**Comment by GiaHan06** [comment]
t lên yutube xem thử ngta làm cũng ok mà hong biết mình làm có ok giống zạy hong nè

**Reply by HuynhLeBaoTran** [reply]
z mình tự design đún hăm

**Reply by GiaHan06** [reply]
ừa bà, được thì tối nay làm nốt luôn do tui sd lap chị nên mượn hơi khó khăn:((

**Comment by GiaHan06** [comment]
ê nhma trang cuối là diễn giải mấy cái kt với cái thông số gì á bài mình tự design thì mình cũng giải thích mấy cái kt luôn hã :)

**Reply by chaudambaotram** [reply]
t ko hiểu cái thông số là sao

**Reply by HuynhLeBaoTran** [reply]
hình như phải v á bà:(((

**Reply by GiaHan06** [reply]
> t ko hiểu cái thông số là sao

thông số là đại lượng đặc trưng cho một tính chất nào đó của một quá trình, hiện tượng, hệ thống, thiết bị kĩ thuật
ai biet gi dau :'(

**Reply by chaudambaotram** [reply]
> > t ko hiểu cái thông số là sao
> 
> thông số là đại lượng đặc trưng cho một tính chất nào đó của một quá trình, hiện tượng, hệ thống, thiết bị kĩ thuật ai biet gi dau :'(

vậy để sau hỏi thầy thử hẻ?Chứ t ko hiểu rồi

**Reply by LTGHan** [reply]
:)))

**Reply by GiaHan06** [reply]
> hình như phải v á bà:(((

*đã thả sad

**Comment by GiaHan06** [comment]
ê lên cái colorhunt.co á kím màu đi nó có mấy cái palettes á

**Reply by chaudambaotram** [reply]
lấy cái ttím mộng mơ hem :)) T thấy thích nó

**Reply by GiaHan06** [reply]
@chaudambaotram @LTGHan @NguyenNgocQuynhAnh @HuynhLeBaoTran ưng màu nào thì gửi mấy cái mã của cái palette đó á tại ngta đã mix cho mình gòi

**Comment by GiaHan06** [comment]
vậy chốt là tự design hã??

**Reply by chaudambaotram** [reply]
ukm t cz mún tụ design nhưng máy cũ quá ko cho :))

**Reply by GiaHan06** [reply]
tối t xin mượn máy chị ròi :))

**Reply by chaudambaotram** [reply]
he he uki

**Reply by LTGHan** [reply]
Kể cho nghe, bữa t mang máy đi diệt virus, mang dìa thấy mất tiu word klm

**Reply by GiaHan06** [reply]
định sẽ call ggmeet nhma cái của tuần 3 t vẫn chưa đăng lên được omggg
mà bữa 37p tận 500 mấy MB h làm mấy tiếng hoi chắc lag máy luôn á :(

**Reply by GiaHan06** [reply]
> Kể cho nghe, bữa t mang máy đi diệt virus, mang dìa thấy mất tiu word klm

gì vậy trời :( bắt đền ngta đi kiu ngta đền máy mới điiii

**Reply by GiaHan06** [reply]
ê nhiều lúc muốn thả sad mà hong có icon sad troi oi

**Comment by LTGHan** [comment]
https://colorhunt.co/palette/fcd1d1ece2e1d3e0dcaee1e1

**Reply by LTGHan** [reply]
Này được ko

**Reply by GiaHan06** [reply]
để sem

**Reply by GiaHan06** [reply]
t thấy có cái màu #D3E0DC hơi chìm :(

**Reply by LTGHan** [reply]
M blend màu zo mà :0

**Reply by GiaHan06** [reply]
> M blend màu zo mà :0

hả?? :)

**Comment by GiaHan06** [comment]
t thấy này ok nè
https://colorhunt.co/palette/fcffa6c1ffd7b5deffcab8ff

**Comment by chaudambaotram** [comment]
https://colorhunt.co/palette/ffe6e6ffabe1a685e26155a6

nè t thích cái này

**Reply by LTGHan** [reply]
Đẹp nè, nma màu cuối hơi đậm

**Reply by LTGHan** [reply]
Hoặc mình kiếm palette nào đậm để dùng chữ trắng đi

**Reply by GiaHan06** [reply]
> Hoặc mình kiếm palette nào đậm để dùng chữ trắng đi

t hong biết nữa
theo t thì thấy chữ trắng hong được tại mình hong có cover màu hết cái trang sợ chữ trắng hong thấy được :)

**Reply by LTGHan** [reply]
Chữ trắng í là chữ có màu sáng á
